numpy/core/fromnumeric.py

```diff
@@ -2782,6 +2782,10 @@ def alen(a):
     7
 
     """
+    # NumPy 1.18.0, 2019-08-02
+    warnings.warn(
+        "np.core.fromnumeric.alen is deprecated, use len instead",
```

Comment:
Soprry, should have made a review: I think this should be ```"`np.alen` is deprecated, use `len` instead."```. Nobody will be using the function through `np.core.fromnumeric.alen`, so I think `np.alen` will do?